cli/tauri.js/src/runner.ts

```diff
@@ -162,35 +192,44 @@ class Runner {
             document.head.appendChild(cspTag)
           }
 
-          writeFileSync(path.join(indexDir, 'index.tauri.html'), dom.serialize())
+          writeFileSync(
+            path.join(indexDir, 'index.tauri.html'),
+            dom.serialize()
+          )
           resolve(inlinedAssets)
         }
-      }).on('progress', event => {
+      }).on('progress', (event: string) => {
         const match = event.match(/([\S\d]+)\.([\S\d]+)/g)
         match && inlinedAssets.push(match[0])
       })
     })
   }
 
-  stop () {
+  async stop(): Promise<void> {
     return new Promise((resolve, reject) => {
       this.tauriWatcher && this.tauriWatcher.close()
-      this.__stopCargo().then(resolve)
+      this.__stopCargo()
+        .then(resolve)
+        .catch(e => {
+          console.error(e)
+        })
     })
   }
```

Comment:
This would be cleaner:
```typescript
async stop(): Promise<void> {
    this.tauriWatcher && this.tauriWatcher.close()
    try {
      await this.__stopCargo()
    } catch (e) {
      console.error(e)
    }
  }
```
Note: if we catch the error at this level, we won't catch anything when calling `stop` (c.f. line 30).